packages/firebase_auth/example/android/app/google-services.json

```diff
@@ -10,7 +10,7 @@
       "client_info": {
         "mobilesdk_app_id": "1:297855924061:android:669871c998cc21bd",
         "android_client_info": {
-          "package_name": "com.yourcompany.firebaseauth.example"
+          "package_name": "io.flutter.plugins.firebaseauthexample"
```

Comment:
I'm not sure this will work. Don't you need to add the new package name to the Firebase project, and download a new google-services.json?

Reply:
Reverted.